### PR TITLE
[5.2] mod_submenu php fixes

### PR DESCRIPTION
**Pull request by peter1szalatnay**
### Summary of Changes

Fix an issues where xdebug always stop because asset is always missing from list, and it is just better then having @ for error handling.

### Testing Instructions

Enable xdebug and go to the system page.

### Actual result BEFORE applying this Pull Request

Exception has occurred.
Warning: Undefined array key 1

### Expected result AFTER applying this Pull Request

No errors

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/modules/mod_submenu/src/Menu/Menu.php [hunk 1]
```diff
@@ -100,7 +100,9 @@ public static function preprocess($parent)
 
             // Exclude item with menu item option set to exclude from menu modules
             if ($itemParams->get('menu-permission')) {
-                @list($action, $asset) = explode(';', $itemParams->get('menu-permission'));
+                $parts  = explode(';', $itemParams->get('menu-permission'));
+                $action = $parts[0];
+                $asset  = (\count($parts) > 1) ? $parts[1] : null;
```
Comment (Quy) [inline administrator/modules/mod_submenu/src/Menu/Menu.php]:
```suggestion
                $asset  = $parts[1] ?? null;
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 2d2ddbcfd885b315743d3ffb885e83bf085f0762<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44320">issues.joomla.org/tracker/joomla-cms/44320</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 2d2ddbcfd885b315743d3ffb885e83bf085f0762<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44320">issues.joomla.org/tracker/joomla-cms/44320</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44320">issues.joomla.org/tracker/joomla-cms/44320</a>.</sub>

**Comment by Hackwar** [comment]
Thanks for your contribution!

**Pull request merged**